src/components/Form/Hooks/useForm.ts

```diff
@@ -65,6 +65,11 @@ export default function useForm<Values = any>(
               block: 'nearest',
               ...options,
             });
+            if (node.tagName !== 'INPUT') {
+              const focusableElement = node.querySelector('input');
+              focusableElement?.focus();
+            }
+            node?.focus();
```

Comment:
- we shouldn't need optional chaining for node inside the if block that validates its existence
- is it necesary to fire focus on node if we know its not an input?

```suggestion
            if (node.tagName === 'INPUT') {
              node.focus();
            } else {
              const focusableElement = node.querySelector('input');
              focusableElement?.focus();
            }
```